File: CarCareHub_/CarCareHub.Services/Database/ChatKlijentZaposlenik.cs
using System;
using System.Collections.Generic;

namespace CarCareHub.Services.Database;

public partial class ChatKlijentZaposlenik
{
    public int ChatKlijentZaposlenikId { get; set; }
    public int KlijentId { get; set; }
    public int ZaposlenikId { get; set; }
    public string Poruka { get; set; }
    public bool PoslanoOdKlijenta { get; set; } 
    public DateTime VrijemeSlanja { get; set; }
    public bool Vidljivo { get; set; }
    public virtual Klijent Klijent { get; set; }
    public virtual Zaposlenik Zaposlenik { get; set; }
}

File: CarCareHub_/UI_/flutter_desktop/lib/models/chatKlijentZaposlenik.dart
import 'package:flutter_mobile/models/zaposlenik.dart';
import 'package:flutter_mobile/models/klijent.dart';
import 'package:json_annotation/json_annotation.dart';
part 'chatKlijentZaposlenik.g.dart';

@JsonSerializable()
class chatKlijentZaposlenik {
   final int chatKlijentZaposlenikId;
  final int? klijentId;
  final Klijent? klijent;
  final Zaposlenik? zaposlenik;
  final int? zaposlenikId;
  final String? poruka;
  final bool? poslanoOdKlijenta;
  final DateTime? vrijemeSlanja;
bool? vidljivo;

  


  chatKlijentZaposlenik(this.chatKlijentZaposlenikId, this.vidljivo, this.klijentId,this.zaposlenikId, this.zaposlenik, this.klijent, 
  this.poruka,this.poslanoOdKlijenta,this.vrijemeSlanja);

  
  factory chatKlijentZaposlenik.fromJson(Map<String,dynamic> json) => _$chatKlijentZaposlenikFromJson(json);


  Map<String,dynamic> toJson() => _$chatKlijentZaposlenikToJson(this);
}
